packages/dds/map/src/interfaces.ts

```diff
@@ -113,6 +156,25 @@ export interface ISharedDirectoryEvents extends ISharedObjectEvents {
     ) => void);
 }
 
+/**
+ * Events emitted in response to changes to the directory data.
+ *
+ * ### "containedValueChanged"
+ *
+ * The containedValueChanged event is emitted when a key is set or deleted.  As opposed to the SharedDirectory's
+ * valueChanged event, this is emitted only on the IDirectory that directly contains the key.
```

Comment:
```suggestion
 * The containedValueChanged event is emitted when a key is set or deleted.  As opposed to the {@link @fluidframework/map#SharedDirectory}'s
 * valueChanged event, this is emitted only on the  {@link @fluidframework/map#IDirectory } that directly contains the key.
```

I like links. But this is probably overkill.

Reply:
And actually they might not even get rewritten properly in the HTML output. Ignore.